src/Providers/UserProviders.jsx

```diff
@@ -16,12 +20,21 @@ class UserProvider extends Component {
                 const user = await generateUserDocument(userAuth);
                 this.setState({ user });
             }
+            // TODO: in the future do an authentication route that returns name and email of user?
+            try {
+                const isValidUser = await httpGet(baseEndpoint + "scenarios");
+                this.setState({ isAdmin: isValidUser !== undefined });
```

Comment:
Works for now, in the future we should ideally check if the response code was 403 or not. If it was something like 500, then that doesn't mean the user isn't logged in, just that the backend server is temporarily unavailable.

Reply:
Hmm good point. Will make a mental note of this

Reply:
I added a PR on the backend, now we should be able to check 200 vs 403 response codes.